src/main/java/Event.java

```diff
@@ -0,0 +1,16 @@
+public class Event extends Task{
+    protected String from;
+    protected String to;
+
+    public Event (String description, String from, String to){
```

Comment:
I think it is better stylistically to put a space between the round and curly brackets